projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/package.json

```diff
@@ -59,7 +60,8 @@
     "@jupyterlab/application": "^3.1.0",
     "@jupyterlab/coreutils": "^5.1.0",
     "@jupyterlab/services": "^6.1.0",
-    "@jupyterlab/settingregistry": "^3.1.0"
+    "@jupyterlab/settingregistry": "^3.1.0",
+    "py-to-ts-interfaces": "git+https://github.com/Syndallic/py-to-ts-interfaces#main"
```

Comment:
This looks a bit dangerous. Doesn't it work with the latest version?

https://pypi.org/project/py-to-ts-interfaces/#history